streamlit_sortables/frontend/src/SortableItem.tsx

```diff
@@ -28,7 +29,7 @@ export const SortableItem: FunctionComponent<SortableItemProps> = ((props) => {
     const className = "btn shadow-none sortable-item " + (props.isActive ? "active" : "");
 
     return (
-      <li className={className} ref={setNodeRef} style={style} {...attributes} {...listeners}>
+      <li className={className} {...style(props.inLineStyles[className])} ref={setNodeRef} style={style} {...attributes} {...listeners}>
```

Comment:
this does not work in my environment. `style` indicates the variable defined at l.24

Reply:
Ah, my mistake, did not realise style was defined elsewhere there. Sorted via:

`import { style as StyleGlamor } from "glamor";`
`{...StyleGlamor(props.inLineStyles[className])}`